frontend/store/projects.js

```diff
@@ -160,11 +160,11 @@ export const getters = {
       text: 'CSV'
     }
     const json = {
-      type: 'json',
+      type: 'jsonl',
```

Comment:
This changes the path in `views.py:TextDownloadAPI()` and wrongly executes `JSONPainter.paint_labels()`, thus resulting in error 500 for the respective project types.

We need a better way to differentiate multiple "jsonl" formats or stick with "json" here.